src/components/StatVideoCard.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react'
+
+export default function StatVideoCard({video}) {
+    const [url, setUrl] = useState(video.url)
```

Comment:
This state is for showing the url not setting it. So also add a button that has the onClick effect of changing to True and then show the div with the url